builds/jazz-build-module/utility-loader.groovy

```diff
@@ -171,4 +171,30 @@ def isReplayedBuild() {
   currentBuild.rawBuild.getCauses().any{ cause -> cause.toString().contains(replayClassName) }
 }
 
+/*
+* Get the required account
+*/
+def getRequiredData(service_config){
```

Comment:
rename to getAccountInfo or something meaningful that gives more context on what is being retrieved.

Reply:
Reverted back the name to getAccountInfo and getAccountInfoPrimary